docs/topics_5/web-execution-config.adoc

```diff
@@ -44,10 +44,10 @@ Project custom rules are maintained using the controls below.
 The *Add* button allows the user to select the rules files ([x-]`.windup.xml` or [x-]`.mta.xml`).
 Selected (checked) rules are shown in the right-hand panel and will be executed during the analysis.
 
-.Custom Rules
-image::web-custom-rules-project.png[The Custom Rules dialog]
+.Custom rules
+image::web-custom-rules-project.png[The custom rules dialog]
 
```

Comment:
Change image name to "Custom rules"" so that it matches its caption.

Reply:
Done. And checked against screenshot.

Reply:
But why is the image name "The custom rules dialog"? THat does not sound good.